src/RoadRegistry.Editor.Schema/Migrations/EditorContextModelSnapshot.cs

```diff
@@ -54,6 +55,21 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("GradeSeparatedJunction","RoadRegistryEditor");
                 });
 
+            modelBuilder.Entity("RoadRegistry.Editor.Schema.MunicipalityGeometryRecord", b =>
+                {
+                    b.Property<string>("NisCode")
+                        .HasColumnType("nvarchar(450)");
```

Comment:
I think NISCodes for municipalities are a lot shorter - https://nl.wikipedia.org/wiki/NIS-code. Where is 450 coming from?